deploy/docs/v1_conf_examples.md

```diff
@@ -0,0 +1,130 @@
+# Configuring Fluentd
+
+Until now, Helm users have not been able to modify their Fluentd configuration outside of the specific parameters that we exposed in the `values.yaml` file. Now, we expose the ability to modify the Fluentd configuration as needed. 
+
+Some use-cases include : 
+ - custom log pipelines, 
+ - adding Fluentd filter plugins (ex: fluentd throttle plugin), or 
+ - adding Fluentd output plugins (ex: forward to both Sumo and S3)
+ - additinal configurantion for sumologic output plugin
+
+
+Below you can see a few examples of how this configuration can be set.
+
+### Custom Log Pipelines
+Now we have exposed an `extraLogs` paramter inside the `logs.containers` section of the `values.yaml` where you can add the output plugin for the custom log pipeline.
```

Comment:
This Fluentd config is enough assuming FluentBit is sending the correct logs, we might want to mention that here, that this will only send the logs to Sumo if the logs are being collected correctly at the FluentBit level with an input plugin.